.github/workflows/sca.yml

```diff
@@ -119,3 +119,23 @@ jobs:
         uses: fossas/fossa-action@main
         with:
           api-key: ${{ secrets.FOSSA_API_KEY }}
+
+  # 5. Validation
+  validation:
```

Comment:
Are we doubling up the linting etc. now by doing it both here and in lint.yml as validate:all does all of them now? Not sure if we just bin lint.yml or split them out at all. I guess validation step is a bit vague for all the steps now happening within here? The output might be quite messy so I'm a bit worried about reading the output of errors if it's one huge step

Reply:
Good point, have removed the `validate:all` from `sca.yml` and updated with `type-check:all` and `validate:yml` in `lint.yml` GHA file.